const.go

```diff
@@ -56,6 +57,15 @@ const (
 	KeySQLInfoRowCount         = 0x00
 	KeySQLInfoAutoincrementIds = 0x01
 
+	keyErrorStack   = 0x00
+	keyErrorType    = 0x00
+	keyErrorFile    = 0x01
+	keyErrorLine    = 0x02
+	keyErrorMessage = 0x03
+	keyErrorErrno   = 0x04
+	keyErrorErrcode = 0x05
+	keyErrorFields  = 0x06
```

Comment:
What about move this constants to box_error.go? These constants don't seem to be used anywhere else.

Reply:
Moved